src/adapters/scene-stream-access-manager.ts

```diff
@@ -95,13 +96,37 @@ export async function createSceneStreamAccessManagerComponent({
     return result.rowCount > 0 && result.rows[0].streaming
   }
 
+  async function getActiveStreamings(): Promise<Pick<SceneStreamAccess, 'created_at' | 'ingress_id'>[]> {
```

Comment:
Should it be named:
```suggestion
  async function getExpiredStreamAccesses(): Promise<Pick<SceneStreamAccess, 'created_at' | 'ingress_id'>[] {
```
?